Answers.md

```diff
@@ -0,0 +1,19 @@
+If you saw this HTML: <div class="box box1 box2 box3"></div> which class has the most specificity weight? 
+
+I would say that all 3 box1, box2, and box3 have the equal and the most specificity. 
```

Comment:
There's a sense in which they're on the same level, but the important thing to remember is that whichever class is given last is the most specific. Each subsequent class listed will overwrite any values that were given in the class before.